homeassistant/components/tami4/config_flow.py

```diff
@@ -82,8 +82,11 @@ async def async_step_otp(
                 _LOGGER.exception("Unexpected exception")
                 errors["base"] = "unknown"
             else:
+                name=api.device_metadata.name
+                if name is None:
+                    name = 'tami4'
```

Comment:
Should this string maybe be pulled from translation?

Reply:
from const?